tests/build.rs

```diff
@@ -331,7 +331,7 @@ fn it_builds_with_webpack_name_output() {
     build_creates_assets(&fixture, vec!["script.js"]);
 
     let out = fs::read_to_string(fixture.get_output_path().join("script.js")).unwrap();
-    assert!(out.contains(r#"//# sourceMappingURL=worker.js.map{"version":3,"file":"worker.js""#));
```

Comment:
why did you need to change the test?

Reply:
I think the output assertion is wrong because it contains a JSON file that is not parsed correctly as a JS file and throws an expected token error.

Here's an incorrectly generated script file and the attempt to publish it:

<img width="709" alt="Screenshot 2021-01-13 at 14 00 59" src="https://user-images.githubusercontent.com/19737508/104461994-f0a24d80-55a7-11eb-945d-96857b441d4f.png">
<img width="1227" alt="Screenshot 2021-01-13 at 14 01 10" src="https://user-images.githubusercontent.com/19737508/104461986-ee3ff380-55a7-11eb-8322-6511b68d7773.png">

Btw, this is the first time I've dealt with Rust 🙈

Reply:
Yes, it looks like that test is wrong, based on what it should look like: https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map